### DISCUSSION TITLE
Particle Dissolution/ Use as suspension

### DISCUSSION THREAD

**Discussion by sariiiiii83** [Open-Systems-Pharmacology/Forum#914] [Questions & Problems]
Dear all, 
I am trying to reproduce a PBPK model from the paper “A physiologically based pharmacokinetic and pharmacodynamic (PBPK/PD) model of the histone deacetylase (HDAC) inhibitor vorinostat for pediatric and adult patients and its application for dose specification” by Moj et al. The model was distributed on this forum here: https://github.com/Open-Systems-Pharmacology/Forum/issues/102 @DanielMoj.
I have been able to reproduce intravenous adult model, however I’m having several problems with the oral adult model. Particle Dissolution was used as Formulation and I have some questions about this type of formulation
1-	What does “Number of bins” mean?
2-	In their model in MoBI, when Particle Dissolution is described, the option “Use as suspensión” appears at the end, but when I define Particle Dissolution in PK-SIM, the “Use as suspension” option does not exist. Is this problem due to a different PK-SIM version? In that case, is there any way to include “Use as suspension” option?
Thank you in advance!
![pdduda_1](https://user-images.githubusercontent.com/80781094/121496747-0abb4d00-c9db-11eb-8036-94e46bdf852d.png)
![pdduda2_1](https://user-images.githubusercontent.com/80781094/121496754-0c851080-c9db-11eb-8e87-1f4fc3fcb310.png)

**Comment by AndreDlm** [comment]
Dear @sariiiiii83,

The parameter `Number of  bins` defines how many bins of particle sizes your tablet will be composed of; each bin will have its own particle size. More information on the particle dissolution model can be found in the OSP manual [here](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-formulations#particle-dissolution) and in the original paper [here](https://pubmed.ncbi.nlm.nih.gov/20554023/).
The parameter `Use as suspension` is not used in the particle dissolution model. You could check in the MoBi file whether it's used for any Formula in the Events Building Block but I wouldn't assume so...

André
